usefulUtilities/sit/sitClient.js

```diff
@@ -446,57 +439,66 @@
         canClickOnModelToSit: false
     };
     function updateUserData() {
-        var properties = Entities.getEntityProperties(_this.entityID);
-        try {
-            _this.userData = JSON.parse(properties.userData);
-        } catch (e) {
-            console.log("Issue parsing userData" + e);
-        }
+        var userData = Entities.getEntityProperties(_this.entityID).userData;
 
-        if (!_this.userData || _this.userData.canClickOnModelToSit === undefined) {
+        if (!userData || _this.userData.canClickOnModelToSit) {
```

Comment:
Is this supposed to rewrite user data for canClickOnModelToSit to false every time?

Reply:
No. This code is a bit of a mess, lemme fix it up